src/app/map/services/esri-services/tool-service/strategies/drawing/esri-polygon-drawing.strategy.ts

```diff
@@ -20,4 +20,8 @@ export class EsriPolygonDrawingStrategy extends AbstractEsriDrawingStrategy<Draw
     this.sketchViewModel.polygonSymbol = polygonSymbol;
     this.tool = polygonType;
   }
+
+  public override updateInternals(): void {
+    // noop
```

Comment:
Why not implement this `// noop` on the abstract class, and override it where necessary?